Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Fix block-drbd xen script for use with the xl stack
 The new xl stack treats all devices as 'phy'. We fix the block-drbd script to
 allow phy disks.
 .
 Upstream is already aware of this issue:
 http://lists.linbit.com/pipermail/drbd-user/2014-April/020740.html

Last-Update: 2014-06-30
Forwarded: not-needed
--- a/scripts/block-drbd
+++ b/scripts/block-drbd
@@ -250,7 +250,7 @@
     fi
 
     case $t in 
-      drbd)
+      drbd|phy)
         drbd_resource=$p
         drbd_role="$(drbdadm role $drbd_resource)"
         drbd_lrole="${drbd_role%%/*}"
@@ -278,7 +278,7 @@
 
   remove)
     case $t in 
-      drbd)
+      drbd|phy)
         p=$(xenstore_read "$XENBUS_PATH/params")
         drbd_resource=$p
         drbd_role="$(drbdadm role $drbd_resource)"
